vnext/ReactCopies/overrides.json

```diff
@@ -4,1071 +4,1087 @@
       "type": "copy",
       "file": "IntegrationTests\\AccessibilityManagerTest.js",
       "baseFile": "IntegrationTests\\AccessibilityManagerTest.js",
-      "baseVersion": "0.61.5",
-      "baseHash": "1232dce43f286df6f60e3aee70dbc22a4edaba4e",
+      "baseVersion": "0.62.0-rc.3",
+      "baseHash": "7e9b6ece04612d18f0a8bf7e8b7cecae815facd9",
       "issue": 4054
     },
     {
       "type": "copy",
       "file": "IntegrationTests\\AppEventsTest.js",
       "baseFile": "IntegrationTests\\AppEventsTest.js",
-      "baseVersion": "0.61.5",
-      "baseHash": "cd4a4f3b2f2dd010afa1cd7c77b73599ca2189b6",
+      "baseVersion": "0.62.0-rc.3",
+      "baseHash": "f4ba0683d08a550ee8af612519740a8b36e704ed",
       "issue": 4054
     },
     {
       "type": "copy",
       "file": "IntegrationTests\\AsyncStorageTest.js",
       "baseFile": "IntegrationTests\\AsyncStorageTest.js",
-      "baseVersion": "0.61.5",
-      "baseHash": "d37101fcfa67d62a95b2c66a32bb3e820d983ebe",
+      "baseVersion": "0.62.0-rc.3",
+      "baseHash": "363ad4689a7058e7ab525d1b3243390377c33c17",
       "issue": 4054
     },
     {
       "type": "copy",
       "file": "IntegrationTests\\GlobalEvalWithSourceUrlTest.js",
       "baseFile": "IntegrationTests\\GlobalEvalWithSourceUrlTest.js",
-      "baseVersion": "0.61.5",
-      "baseHash": "ef4b23fc1fc049e84257bba1da6ba0d5d216fc66",
+      "baseVersion": "0.62.0-rc.3",
+      "baseHash": "57af3fe84649f53262ea8503004dc1bfea4011ca",
       "issue": 4054
     },
     {
       "type": "copy",
       "file": "IntegrationTests\\ImageCachePolicyTest.js",
       "baseFile": "IntegrationTests\\ImageCachePolicyTest.js",
-      "baseVersion": "0.61.5",
-      "baseHash": "14ba7b115a8725987b06b2804c331140b124dc25",
+      "baseVersion": "0.62.0-rc.3",
+      "baseHash": "6e77238435d9a4d4d299ae0082bdf6c69d83138b",
       "issue": 4054
     },
     {
       "type": "copy",
       "file": "IntegrationTests\\ImageSnapshotTest.js",
       "baseFile": "IntegrationTests\\ImageSnapshotTest.js",
-      "baseVersion": "0.61.5",
-      "baseHash": "ff34a4d19938aada60af66d65773c89c8c70eae0",
+      "baseVersion": "0.62.0-rc.3",
+      "baseHash": "f12b95ca9839786a9f1412a40437b4bf563d3bbd",
       "issue": 4054
     },
     {
       "type": "copy",
       "file": "IntegrationTests\\IntegrationTestHarnessTest.js",
       "baseFile": "IntegrationTests\\IntegrationTestHarnessTest.js",
-      "baseVersion": "0.61.5",
+      "baseVersion": "0.62.0-rc.3",
```

Comment:
>baseVersion [](start = 7, length = 11)

A bunch of files (like this one) didn't actually change. It was easier to just dump a new copy of stuff in though and regenerate the manifest.